test-scenarios/ovn-low-scale.yml

```diff
@@ -3,6 +3,7 @@ global:
 
 cluster:
   clustered_db: true
+  enable_ssl: False
```

Comment:
This seems unrelated, please revert this.

Reply:
> This seems unrelated, please revert this.

its very related for ovs-etcd we need to use tcp protocol since the client only listen to tcp on port 6641